changelog/+schema-set-cache.added.md

```diff
@@ -0,0 +1 @@
+Add method schema.set_cache() to populate the cache manually (primarily for unit testing)
```

Comment:
I've typically written this as `client.schema` instead of just `schema`. Don't know which is best or easiest to understand.